finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -15,6 +17,17 @@ private[netty4] class ServerBridge[In, Out](
 
   def initChannel(ch: Channel): Unit = {
     val transport: Transport[In, Out] = transportFac(ch)
-    serveTransport(transport)
```

Comment:
Interesting. I'm pretty sure SslHandler should work just fine with auto-read is disabled. Maybe it's b/c we're not issuing an initial `read` given that `ChannelTransport` is not instantiated? If so I think it's a bug in Netty and we should file a ticket about that. As a workaround in your "delay" handler you can issue a `read` manually.